cv_next/components/ui/filters/valueSelect.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react"
+
+export const DropdownInput = ({
+    placeHolder,
+    valueIds,
+    valueId,
+    onChange,
+    getValueById,
+    text
+  }: {
+    placeHolder: string,
+    valueIds: number[],
+    valueId: number | null,
+    getValueById: (id: number) => string
+    onChange: (newValue: number | null) => void,
+    text: string
+  }) => {
+    const [isMenuOpen, setIsMenuOpen] = useState(false)
+    const isPlaceHolder = valueId === null
+
+    const changeIsMenuOpen = () => {
+        setIsMenuOpen(!isMenuOpen)
+    }
+
+    return <>
```

Comment:
You have multiple conditional rendering flags in an already hard-to-digest tw block of code.
Please separate them as constants above.
e.g. `const menuOpenStyle = isMenuOpen ? "outline-b-0 rounded-b-none" : "";`, then using it later as simply `{menuOpenStyle}`.
Code readability is important, especially with tw classes :)